app/src/main/java/com/artemzin/qualitymatters/ApplicationModule.java

```diff
@@ -33,6 +34,11 @@ public QualityMattersApp provideQualityMattersApp() {
     }
 
     @Provides @NonNull @Singleton
+    public Context provideContext() {
```

Comment:
We already provide `QualityMattersApp`, please use it instead of such general thing as `Context` :)